angular/src/app/app.component.html

```diff
@@ -1,7 +1,37 @@
-<!--The content below is only a placeholder and can be replaced.-->
 <div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
+	<h1>Welcome to {{ title }} ! </h1>
 </div>
-<app-correlation-graph></app-correlation-graph>
+<div class="container">
+	<div class="row">
```

Comment:
It would be nice to have the navigation within a component

Reply:
fixed